### PR TITLE
Fix #77, Fix #131

### LINKED ISSUE #77: [Windows] Double Clicking DragToResizeArea should Simulate Vertical Aero Snap
Double clicking the top and bottom part of  DragToResizeArea widget doesn't do anything when it should've maximized the window vertically and could be restored to the original size.
According to this
https://stackoverflow.com/questions/34230743/how-to-vertically-resize-a-window-in-code
SetWindowPlacement is the way to do it

### LINKED ISSUE #131: window overflow
calling 

```
      windowManager.setFullScreen(true);
```
      
      
then going back to normal callling 
```
  windowManager.setFullScreen(false);
      windowManager.restore();
```

cases window to go out of boundry in the bottom part 

normal view 
image

after calling the commands
image

using window_manager 0.2.2

and it was happening on 0.2.1 too

### PR DESCRIPTION
**Pull request by damywise**

### CODE REVIEW

File: windows/window_manager_plugin.cpp [hunk 1]
```diff
@@ -465,11 +471,6 @@ void WindowManagerPlugin::HandleMethodCall(
         std::get<flutter::EncodableMap>(*method_call.arguments());
     window_manager->SetProgressBar(args);
     result->Success(flutter::EncodableValue(true));
-  } else if (method_name.compare("setIcon") == 0) {
```
Comment (lijy91) [inline windows/window_manager_plugin.cpp:468]: Why was this removed

File: windows/window_manager.cpp [hunk 2]
```diff
@@ -714,27 +756,31 @@ void WindowManager::StartResizing(const flutter::EncodableMap& args) {
   bool bottom = std::get<bool>(args.at(flutter::EncodableValue("bottom")));
   bool left = std::get<bool>(args.at(flutter::EncodableValue("left")));
   bool right = std::get<bool>(args.at(flutter::EncodableValue("right")));
+  bool isDoubleClick =
```
Comment (lijy91) [inline windows/window_manager.cpp:759]: I think we should use a new method to implement this functionality instead of increasing it in startResizing

File: windows/window_manager_plugin.cpp (hunk 1 above)
Comment (damywise) [reply windows/window_manager_plugin.cpp:468]: Ah, sorry this commit was pretty old that it wasn't implemented yet, and I didn't check it properly. I'll add it later.

File: windows/window_manager.cpp (hunk 2 above)
Comment (damywise) [reply windows/window_manager.cpp:759]: Alright

Comment (damywise) [reply windows/window_manager.cpp:759]: Should I make it private or public?

**Comment by lijy91** [comment]
I think these two issues should be split into two PRs to fix. For #77 I might prefer to add a vertically parameter to the maximize method
